projects/igniteui-angular/src/lib/grids/columns/column-group.component.ts

```diff
@@ -26,6 +26,54 @@ export class IgxColumnGroupComponent extends IgxColumnComponent implements After
 
     @ContentChildren(IgxColumnComponent, { read: IgxColumnComponent })
     children = new QueryList<IgxColumnComponent>();
+
+    /**
+     * Set if the column group is collapsible.
+     * Default value is `false`
+     * ```html
+     *  <igx-column-group [collapsible] = "true"></igx-column-group>
+     * ```
+     * @memberof IgxColumnGroupComponent
+     */
+    @Input()
+    public set collapsible(value: boolean) {
+        this._collapsible = value;
+        this.collapsibleChange.emit(this._collapsible);
+        if (this.children && !this.hidden) {
+            if (value) {
```

Comment:
```suggestion
            if (this._collapsible) {
```
A listener may change the value of `_collapsible` on emit.